packages/react-core/src/components/Banner/Banner.tsx

```diff
@@ -3,16 +3,20 @@ import styles from '@patternfly/react-styles/css/components/Banner/banner';
 import { css } from '@patternfly/react-styles';
 
 export interface BannerProps extends React.HTMLProps<HTMLDivElement> {
-  /** Content rendered inside the banner */
+  /** Content rendered inside the banner. */
   children?: React.ReactNode;
-  /** Additional classes added to the banner */
+  /** Additional classes added to the banner. */
   className?: string;
-  /** Variant styles for the banner */
-  variant?: 'default' | 'info' | 'danger' | 'success' | 'warning';
   /** If set to true, the banner sticks to the top of its container */
   isSticky?: boolean;
-  /** Text announced by screen readers to indicate the type of banner. Defaults to "${variant} banner" if this prop is not passed in */
-  screenReaderText?: string;
+  /** Text announced by screen readers to indicate the type of banner. Defaults to "${variant} banner"
+   * if this property is not passed in.
+   *
+   * Pass in null to omit this text from the banner when the banner does not convey status/severity.
+   */
+  screenReaderText?: string | null;
```

Comment:
I am not really a fan of using null here.  Can we instead check the variant value at line 39.  If it is `default`, don't show the text.

Reply:
We could, though it would still render status/severity text for a banner that may only be using a "success" variant for the green color rather than for status for example. The `null` check was essentially just a way to allow selecting a variant for the color only, similar to how the [Core banner examples](https://www.patternfly.org/v4/components/banner/html) are now setup.

Michael did open issues for replacing the status variant with color (https://github.com/patternfly/patternfly-react/issues/8204), though that's for the breaking change. If we'd want to wait for that to fix that up I could implement this suggestion of only rendering the SR text for non-default variants, otherwise we could introduce a new prop (`color` as mentioned in the original comment, or maybe a `hasStatus` to determine whether sr text gets rendered).

Reply:
That makes sense. So rather than on line 39 having `screenReaderText || `${variant} banner``, instead we'd just want `screenReaderText`, and only render that `pf-u-screen-reader` element if the prop is passed in. If that's the case I'll update the logic + examples

Reply:
@thatblindgeye Yes